src/pages/partners/partners.module.scss

```diff
@@ -0,0 +1,27 @@
+.introduction {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: center;
+  gap: 2rem;
+  margin-top: 4rem;
+  margin-bottom: 4rem;
+}
+
+.intro_quote {
+  width: 60%;
+  line-height: 25px;
+  font-style: italic;
+  font-weight: 400;
+  font-size: 22px;
```

Comment:
width: 60% will have empty space when it wraps on small screens. I would use flex-basis of an appropriate width (can be 60% or some value like 50ch) with flex grow.

As for the font styling a bigger line height looks better imo. The text could be centered vertically with align-self: center as well